### DISCUSSION TITLE
[Question]: Is it possible to attach the preview.gif to HA notification?

### DISCUSSION THREAD

**Discussion by luisiam** [blakeblackshear/frigate#13220] [Ask A Question] [answered]
### What is your question:

When I check the documentation, only thumbnail, snapshot and clips are available to HA notification.

`
https://HA_URL/api/frigate/notifications/<event-id>/thumbnail.jpg
`
`
https://HA_URL/api/frigate/notifications/<event-id>/snapshot.jpg
`
`
https://HA_URL/api/frigate/notifications/<event-id>/clip.mp4
`

Is the preview.gif available as well and what will be the URL? I noticed that it is available in the HTTP API but my Frigate instance is not exposed to the internet.

`
GET /api/events/<id>/preview.gif
`

**Comment by NickM-27** [comment]
Yes it is possible, it was added in integration 5.3.0

**Reply by luisiam** [reply]
Is it the image entity of the detected object? Is there a direct URL I can use like the thumbnail/snapshot/clip?

**Reply by NickM-27** [reply, marked as answer]
https://HA_URL/api/frigate/notifications/<event-id>/event_preview.gif

**Reply by luisiam** [reply]
Great! Thanks you!
